assets/css/app.css

```
@@ -10,8 +10,8 @@
 @theme {
     --font-mono: "Operator Mono Lig";
         --font-display: "Formula Condensed";
-        --font-body: "Product Sans";
-            --font-sans: "Product Sans";
+        --font-body: "DM Sans", ui-sans-serif, system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)